### PR TITLE
Bg add footer

### PR DESCRIPTION
**Pull request by BhavyaGulati**
Closes #X

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [ ] I have replaced `X` above with the associated issue number.
- [ ] I have changed the issue's label to "pull request".
- [ ] I have assigned the designated reviewer to this PR.
- [ ] I have set the appropriate label on this PR.
- [ ] I have written clear commit messages ([ref](http://chris.beams.io/posts/git-commit/)).

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

**Comment by AradhyaGupta** [comment]
[(Bhavya)React Issue 1: Add Footer](https://trello.com/c/SGTzAi3F/70-bhavyareact-issue-1-add-footer)

File: app/controllers/api/feedbacks_controller.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module Api
   class FeedbacksController < ApplicationController
     def create
-     # Implement your create action
+      # Implement your create action
```
Comment (jmangel) [inline app/controllers/api/feedbacks_controller.rb]: This change is unrelated, can we drop it?

File: app/javascript/image_sharing/test/components/App.spec.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  it('should render sub-components', () => {
+    const wrapper = mount(
+      <App stores={[]}/>
+    );
+
+    expect(wrapper.find('Header')).to.have.lengthOf(1);
+    expect(wrapper.find('Header').text()).equal('Tell us what you think');
+    expect(wrapper.find('Footer')).to.have.lengthOf(1);
+    expect(wrapper.find('Footer').text()).equal('Copyright: AppFolio Inc. Onboarding');
```
Comment (jmangel) [inline app/javascript/image_sharing/test/components/App.spec.js]: Since we're testing this in the `Footer.spec.js`, we can probably remove the text test here. We still want to test that the footer is there (like in line 12), but if someone makes a change to the wording in the Footer, it would be nice if they only have to change the corresponding test file.

File: app/controllers/api/feedbacks_controller.rb (hunk 1 above)
Comment (BhavyaGulati) [reply app/controllers/api/feedbacks_controller.rb]: sure

Comment (butzhang) [inline app/controllers/api/feedbacks_controller.rb]: can we remove this line?

File: app/javascript/image_sharing/test/components/App.spec.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import { expect } from 'chai';
+import App from '../../components/App';
+import Adapter from 'enzyme-adapter-react-16';
+
+describe('<App />', () => {
+  configure({ adapter: new Adapter() });
+
+  it('should render sub-components', () => {
+    const wrapper = mount(
+      <App stores={[]}/>
```
Comment (butzhang) [inline app/javascript/image_sharing/test/components/App.spec.js]: why we need to pass `[]` into the stores `props`?

File: app/views/feedbacks/new.html.erb [hunk 4]
```diff
@@ -1 +1,3 @@
+<% stylesheet_link_tag "/Users/bhavya.gulati/Documents/image-sharer-BhavyaGulati/app/assets/stylesheets/images.css" %>
```
Comment (butzhang) [inline app/views/feedbacks/new.html.erb]: we don't use full path here. And can you try use react only to render style? here is the [ref link](https://reactjs.org/docs/dom-elements.html#style)

**Comment by butzhang** [comment]
The deployment also failed. You have to fix it in order to proceed.

File: app/controllers/api/feedbacks_controller.rb (hunk 1 above)
Comment (BhavyaGulati) [reply app/controllers/api/feedbacks_controller.rb]: It's added by rubocop and circle CI fails due to unresolved rubocop issues if I don't add this space

File: app/javascript/image_sharing/test/components/App.spec.js (hunk 3 above)
Comment (BhavyaGulati) [reply app/javascript/image_sharing/test/components/App.spec.js]: If I remove it, I get following error
```  11 |   it('should render sub-components', () => {
  12 |     const wrapper = mount(
> 13 |       <App stores={}/>
     |                   ^
  14 |     );
  15 | 
  16 |     expect(wrapper.find('Header')).to.have.lengthOf(1);
    at Parser.pp$5.raise (/Users/bhavya.gulati/Documents/image-sharer-BhavyaGulati/node_modules/babylon/lib/index.js:4454:13)
```

File: app/views/feedbacks/new.html.erb (hunk 4 above)
Comment (BhavyaGulati) [reply app/views/feedbacks/new.html.erb]: Thanks for sharing! added this refactoring in second commit
